conf/docker/dataverse-glassfish/startup

```diff
@@ -0,0 +1,863 @@
+#!/usr/bin/perl
```

Comment:
How is `startup` different than `install`? We don't want to maintain both of these, do we?